src/components/__snapshots__/Image.test.js.snap

```diff
@@ -7,7 +7,7 @@ exports[`renders correctly 1`] = `
   source={
     Object {
       "height": 357,
-      "id": "3CU8n6O760GJ80Wr7r",
+      "id": "4D776SGAHYRnNGHx1DES622EFYZ",
```

Comment:
If we update to Jest 23 we can avoid these sorts of snapshot changes where the value is unimportant by using [property matchers](https://facebook.github.io/jest/blog/2018/05/29/jest-23-blazing-fast-delightful-testing.html#snapshot-property-matchers)

Reply:
Interesting. I will take a look. Do you think it is worthwhile looking now or as something after this refactor?

Reply:
It shouldn't block this but definitely check it out.